dask/dataframe/tests/test_dataframe.py

```diff
@@ -1821,3 +1821,19 @@ def test_columns_assignment():
     ddf[['a', 'b']] = ddf2[['y', 'z']]
 
     eq(df, ddf)
+
+
+@pytest.mark.parametrize("idx", [
+    np.arange(100),
+    sorted(np.random.random(size=100)),
+    pd.date_range('20150101', periods=100)
+])
```

Comment:
Good point, plus all I had done was test the default behavior when I really needed to test `skipna=False`.  I don't think `parametrize` tests over all combinations of inputs, just a list of inputs.  Would be a cool feature to have though.  I just added a for loop.